frontend/pages/settings.tsx

```diff
@@ -31,11 +35,18 @@ const Notification = styled.span`
 `
 
 type SettingsProps = {
-  userInfo: UserInfo
+  userInfo?: UserInfo
   authenticated: boolean | null
+  submissions: ApplicationSubmission[]
+  whartonapplications: any
```

Comment:
Nit: camel case variable name and adding types would be nice (I know the reference code didn't have either but it would be good practice)